reporting_app/static/datatable.js

```diff
@@ -92,6 +92,18 @@ var merge_multi_sources_keep_first = function(dt_config){
     return _merge_multi_sources(dt_config, merge_on_keep_first);
 }
 
+var test_exist = function(variable){
+    if ( variable instanceof Array ) {
+        return variable.length > 0;
+    }
+    return variable !== undefined && variable !== null && variable;
+}
+
+var color_filter = function( row, data, dataIndex ) {
+    if ( test_exist(data["trim_r1"]) || test_exist(data["trim_r2"]) || test_exist(data["tiles_filtered"]) ) {
```

Comment:
We should probably make this a configurable list of column IDs that can be registered for colouring